packages/adapter-components/src/definitions/system/fetch/resource.ts

```diff
@@ -37,9 +37,9 @@ type ConfigSuggestion = {
   value: ConfigChangeSuggestion
 }
 
-type OnErrorHandlerAction = FailEntireFetch | CustomSaltoError | ConfigSuggestion
+export type OnErrorHandlerAction = FailEntireFetch | CustomSaltoError | ConfigSuggestion
 
-type OnErrorHandler = ArgsWithCustomizer<OnErrorHandlerAction, OnErrorHandlerAction, Error>
+type OnErrorHandler = ArgsWithCustomizer<OnErrorHandlerAction, OnErrorHandlerAction, { error: Error, typeName: string }>
```

Comment:
What do you think about adding it as the `AdditionalArgs` of the `ArgsWithCustomizer` instead?
Which means:
```suggestion
type OnErrorHandler = ArgsWithCustomizer<OnErrorHandlerAction, OnErrorHandlerAction, Error, { typeName: string }>
```

I'm not sure if it really matters, the naming just makes sense to me. Your call :)

Reply:
I think the difference is `AdditionalArgs` are used to create the `custom` func, so i'd say in this case `typeName` is more like the `custom` function input and not it's creator